ext/gd/libgd/gd_avif.c

```diff
@@ -103,7 +103,7 @@ static avifBool setEncoderTilesAndThreads(avifEncoder *encoder, avifRGBImage *rg
 
 	// The number of tiles in any dimension will always be a power of 2. We can only specify log(2)tiles.
 
-	tilesLog2 = floor(log2(tiles));
+	tilesLog2 = (int) floor(log2(tiles));
```

Comment:
Why do we even `floor()` here instead of casting to int right away:
```suggestion
	tilesLog2 = (int) log2(tiles);
```
And given we're interested in the log2 of an integer, there are even options not requiring an FP instruction (especially since tiles <= 8). But okay.